src/autosuggest/internal.tsx

```diff
@@ -181,6 +181,8 @@ const InternalAutosuggest = React.forwardRef((props: InternalAutosuggestProps, r
     filteringResultsText: filteredText,
   });
 
+  const shouldRenderDropdownContent = !isEmpty || dropdownStatus.content;
```

Comment:
It is possible to add a unit test to cover the change?

Reply:
> This might need to include dropdownStatus.footer. (In the property-filter the empty state is added to the sticky footer).

There is no `dropdownStatus.footer` here. If it `isSticky`, we will pass the `dropdownStatus.content` to the `dropdownFooter` property.

But if I got it right, I made the [change](https://github.com/cloudscape-design/components/pull/1567/files#diff-22d221ed5a882afacd0f403f72e9841f105c338be4766fa47a92ee2ef74907b5R282) to account for this concern: `open={open && (!!dropdownContent || !!dropdownFooter)}`.

> It is possible to add a unit test to cover the change?

Added!